library/ssl_tls13_generic.c

```diff
@@ -721,29 +813,40 @@ int mbedtls_ssl_tls13_process_certificate( mbedtls_ssl_context *ssl )
     int ret = MBEDTLS_ERR_ERROR_CORRUPTION_DETECTED;
     MBEDTLS_SSL_DEBUG_MSG( 2, ( "=> parse certificate" ) );
 
+    /* Coordination:
+     * Check if we expect a certificate, and if yes,
+     * check if a non-empty certificate has been sent.
+     */
 #if defined(MBEDTLS_KEY_EXCHANGE_ECDHE_ECDSA_ENABLED)
-    unsigned char *buf;
-    size_t buf_len;
+    if( ssl->handshake->cert_request_send )
```

Comment:
Not needed I think as we do not reach that state if we did not send a CertificateRequest message.